### ISSUE TITLE
Building PECL grpc-1.29.0 fails on Windows

### ISSUE THREAD

**Issue by cmb69** [grpc/grpc#23002]
The Windows builds of the freshly released PECL grpc-1.29.0 [fail](https://windows.php.net/downloads/pecl/releases/grpc/1.29.0/logs/), because the include path to third_party/boringssl-with-bazel/src/include is not properly set. I've fixed this directly in the shipped config.w32 to test the build locally, but probably the proper fix would be something like:
````.diff
 templates/config.w32.template | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/templates/config.w32.template b/templates/config.w32.template
index f69cc08df65..84589544b7a 100644
--- a/templates/config.w32.template
+++ b/templates/config.w32.template
@@ -33,7 +33,7 @@
       "/I"+configure_module_dirname+"\\src\\php\\ext\\grpc "+
       "/I"+configure_module_dirname+"\\third_party\\abseil-cpp "+
       "/I"+configure_module_dirname+"\\third_party\\address_sorting\\include "+
-      "/I"+configure_module_dirname+"\\third_party\\boringssl\\include "+
+      "/I"+configure_module_dirname+"\\third_party\\boringssl-with-bazel\\src\\include "+
       "/I"+configure_module_dirname+"\\third_party\\upb "+
       "/I"+configure_module_dirname+"\\third_party\\zlib ");
````

**Comment by stanley-cheung** [comment]
Yes aware of that. Already merged #22993. Will push another release soon.

**Comment by stanley-cheung** [comment]
PECL extension version 1.29.1 is live now. Verified the DLLs are built.
